taskcluster/ci/config.yml

```diff
@@ -48,7 +48,6 @@ treeherder:
         feature-intent: feature-intent
         feature-logins: feature-logins
         feature-media: feature-media
-        feature-p2p: feature-p2p
```

Comment:
I don't recall ever adding a line here to the file, so I also wasn't sure if I needed to remove it as well.